project.yml

```diff
@@ -56,7 +56,7 @@ commands:
   - name: create_kb
     help: "Creates KB utilizing SQLite database with Wiki content."
     script:
-      - "env PYTHONPATH=scripts python ./scripts/create_kb.py ${vars.vectors_model} ${vars.language}"
+      - "env PYTHONPATH=scripts python ./scripts/cli_create_kb.py ${vars.vectors_model} ${vars.language}"
     deps:
       - "output/${vars.language}/wiki.sqlite3"
```

Comment:
(commented on the wrong line because GH doesn't let me comment on non-edited lines)

The `clean` command should use `rm -rf` instead of just `rm -f`

Reply:
Updated in https://github.com/explosion/wikid/pull/4/commits/33d6db297cb5d8bf9ee85023c604b8f9e236cd3e.